src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+  justify-content: space-between;
+}
```

Comment:
this is 100% unnecessary styles

Reply:
width: 96px;
 height: 96px;
this help to make it square as on reference